test/basic.test.ts

```diff
@@ -363,8 +362,7 @@ describe('pages', () => {
     // don't expect any errors or warning on client-side navigation
     const { page: page2, consoleLogs: consoleLogs2 } = await renderPage('/')
     await page2.locator('#to-client-only-components').click()
-    // force wait for a few ticks
-    await page2.waitForTimeout(50)
```

Comment:
Yeah i think i wanted to wait for several ticks, I wans't ware of the `waitFor` at that time 😅